### PR TITLE
Ui improvements

### PR DESCRIPTION
**Pull request by smth**

### CODE REVIEW

**Comment by vitorbaptista** [comment]
@smth is this good to go?

**Comment by smth** [comment]
Yup.

**Comment by vitorbaptista** [comment]
@smth The changes look good, there're just a few comments

* Why changing "Search" to "Explore"? I think "Search", being more common, has a clearer meaning;
* There must be a "Contribute Data" (or similar) link, so an anonymous user can add data not necessarily related to a trial. Maybe we could add a text like "Is this related to a trial? If so, please search for the trial and contribute data from there, or give us the trial ID here" (with a better wording), just in case the data is related to a trial;
* Do we really need the OK Ribbon? It looks alien to the rest of the page, and is too prominent for something that probably most of our users won't be interested in (not to mention the extra KBs added)

**Comment by smth** [comment]
I'm just carrying out tasks (https://github.com/opentrials/opentrials/issues/121, https://github.com/opentrials/opentrials/issues/122) from @pwalsh at this point. It sounds like the "Contribute Data" link maybe should have remained though?

My 2¢ on the ribbon thing is, I don't like it (anywhere). I feel this is a bigger conversation that needs to be had though.

**Comment by vitorbaptista** [comment]
@smth OK, so from my comments just re-add the "Contribute Data" link. Part of the requirements for contributing data was that it doesn't need to be related to a trial, so that's still needed.

We'll talk about the ribbon offline :wink:

**Comment by smth** [comment]
[Done](https://github.com/opentrials/opentrials/pull/168/commits/bcc2a37fdbd3713582f5c5ead06505df82893c35)

File: views/partials/main-menu.html [hunk 1]
```diff
@@ -4,6 +4,11 @@
   </a>
 </li>
 <li>
+  <a href="/search">
+    Contribute Data
```
Comment (vitorbaptista) [inline views/partials/main-menu.html]: That should be `/contribute-data`

Comment (smth) [inline views/partials/main-menu.html]: Updated.

**Comment by vitorbaptista** [comment]
Merged :+1:

**Pull request merged**